src/ldclient_eval.erl

```diff
@@ -113,6 +113,26 @@ all_flags_state(Context, Options, Tag) ->
 is_not_deleted(#{deleted := true}) -> false;
 is_not_deleted(_) -> true.
 
+-spec is_prereq_of(FlagKey :: ldclient_flag:key(), Event :: ldclient_event:event()) -> boolean().
+is_prereq_of(FlagKey, #{data := #{prereq_of := PrereqOf}} = _Event) ->
```

Comment:
If prereq_of is set to the flag we are evaluating, then that flag was evaluated as a direct prerequisite.